data/presets/presets/building/apartments.json

```diff
@@ -1,5 +1,8 @@
 {
     "icon": "maki-building",
+    "fields": [
+        "building/flats"
```

Comment:
Oh, you should also have `"{building}"` here the way you did before.

Reply:
So it should be `  "fields": [ "{building}"]` ? Without "flats"?  or ` "fields": [ "{building}","flats"]`  ?

I think I need more time to get more familiar with the code base :)

Reply:
Oh.. I think I figured it out. It should be `"fields": [
        "{building}",
        "building/flats"
    ]`
:)